package.json

```diff
@@ -28,8 +28,7 @@
     "clean": "rimraf dist",
     "format": "prettier --write --loglevel warn \"src/**/*.ts\" \"*.md\"",
     "format:verify": "prettier --check --loglevel warn \"src/**/*.ts\" \"*.md\"",
-    "lint": "tslint --project tsconfig.json --fix",
-    "lint:verify": "tslint --project tsconfig.json",
+    "lint": "eslint -c .eslintrc.js --ext .ts src",
```

Comment:
still seems worth it to split this into a `lint` and `lint:verify`. the `lint` version should have `--fix` and the verify one shouldn't. or is there something i'm missing?